test_data/yam/agama_rmt.yaml

```diff
@@ -0,0 +1,6 @@
+---
+repos:
+  - name: SLE-Product-SLES-%VERSION%
+    alias: SUSE_Linux_Enterprise_Server_%VERSION%_%ARCH%:SLE-Product-SLES-%VERSION%
```

Comment:
apparently the URI changes, for example, https://openqa.suse.de/tests/17710627#step/zypper_lr/3 so that it what we could ensure in the validation.

Reply:
Removed `Draft` and I'd change that to make sure it works after the bug was fixed.